static/js/util/util.js

```diff
@@ -227,44 +228,72 @@ complete profile consists of:
     `currently employed == 'yes'`
 */
 export function validateProfileComplete(profile) {
```

Comment:
Can you undo the changes to this function? this is a lot of extra code that doesn't really give us anything. we always want to redirect the user to the first tab (personal -> education -> professional -> privacy) which has an invalid field on it, so we don't need to check the professional tab if we find something missing on the personal tab (which is why the function previously returned any time it found any validation error).

I think the only thing we really need to change to get the functionality we want is passing in the validation messages.